srcpkgs/python-pywatchman/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'python-pywatchman'
+pkgname=python-pywatchman
+version=1.4.1
+revision=1
+noarch=yes
+wrksrc="watchman-4.9.0"
+build_wrksrc="python"
+build_style=python-module
+pycompile_module="pywatchman"
+hostmakedepends="python-devel python3-devel python-setuptools python3-setuptools"
+makedepends="python-devel python3-devel"
+depends="python watchman"
+short_desc="Connect and query Watchman to discover file changes (Python 2)"
+maintainer="Hoang Nguyen <hoang@wetrust.io>"
+license="BSD-3-Clause"
+homepage="https://github.com/facebook/watchman/tree/master/python"
+distfiles="https://github.com/facebook/watchman/archive/v4.9.0.tar.gz"
```

Comment:
should use ${version}

Reply:
there are 2 different versions here:

The pywatchman version is 1.4.1
The pywatchman however sits in the watchman repo whose version is 4.9.0

I am thinking of declaring another variable called `watchman_version` WDYT?

Reply:
Decided on taking the PYPI tarball approach